business_rules/operators.py

```diff
@@ -282,16 +297,16 @@ def not_exists(self, other_value):
     
     @type_operator(FIELD_DATAFRAME)
     def equal_to(self, other_value):
-        target = other_value.get("target")
-        comparator = other_value.get("comparator")
+        target = self.replace_prefix(other_value.get("target"))
```

Comment:
This seems like it would be great for a pre-processing step where we just clean up the rule and add the prefix everywhere. Adding it to every method seems like it might cause headaches in the future when you need to change something. Is there any limitation on why we can't do it before calling the operations?

Reply:
I did consider this, my main reasoning for doing this is because in the future we will need to know whether or not we are operating on a column or a string. I figured we'd want to add this logic to the engine and because we wouldn't know whether the comparator is meant to be a column name or a string until we get to the operators, I figured it would be best to hold off on updating the provided comparator until we get to the actual operator